docs/manual/readme.md

```diff
@@ -750,43 +750,29 @@ Please note that we require Helm **3+**, and suggest **3.2+**, so be sure to set
 
 Helm chart is responsible only for the deployment of microservices into a Kubernetes cluster. It is not intended for and cannot create any Azure resources. So the installation of the chart requires one to already have Azure resources that are described in Azure Resources part of previous section. Detail of those Azure resources are provided to the chart as configuration parameters and are passed down to microservices as environment variables.
 
-Documentation of the chart lists all required Azure resources and provides steps for manually creating them if needed. It also described details of Azure resources that are required for installation of the chart. It can be found [here](https://github.com/Azure/Industrial-IoT/blob/helm/0.4.0/deploy/helm/azure-industrial-iot/README.md).
+Documentation of the chart lists all required Azure resources and provides steps for manually creating them if needed. It also described details of Azure resources that are required for installation of the chart. It can be found [here](https://github.com/Azure/Industrial-IoT/blob/helm/0.4.1/deploy/helm/azure-industrial-iot/README.md).
 
 #### Installing the Chart
 
 ##### Add Helm Repository
 
 To install the chart first ensure that you have added either one of two Helm repositories where we publish azure-industrial-iot Helm chart. It can be found in:
 
-- <https://azure.github.io/Industrial-IoT/helm>
-
-  > NOTE: This is the recommended Helm chart repository to use.
+- <https://microsoft.github.io/charts/repo>
```

Comment:
Please bring back details about `https://azure.github.io/Industrial-IoT/helm` Helm chart repository, preferably with the notes. Going forward this is the one that clients should use.